Libs/MRML/Core/vtkMRMLDisplayableNode.cxx

```diff
@@ -319,8 +319,9 @@ void vtkMRMLDisplayableNode::SetDisplayClassVisibility(const char* nodeClass, in
 }
 
 //---------------------------------------------------------------------------
-void vtkMRMLDisplayableNode::GetRASBounds(double bounds[6])
+bool vtkMRMLDisplayableNode
+::GetRASBounds(double bounds[6], bool useTransform)
```

Comment:
You cannot change this method, as it is overridden in many MRML nodes in many extensions. Also, the method name clearly states that bounds are returned in RAS coordinate system. You shouldn't add a flag that completely changes the method.

Instead, you could add a new GetBounds(double bounds[6]) method.

bool return value is not necessary, as it is a common VTK convention that a bounds is invalid if any of the bounds are inverted (bool valid = bounds[0]<=bounds[1] || bounds[2]<=bounds[3] || bounds[4]<=bounds[5]). Returning a bool value would be redundant, as you would need to check validity of the bounds aunway.